website/content/en/docs/getting-started/_index.md

```diff
@@ -177,7 +177,13 @@ metadata:
 spec:
   provider:
     instanceProfile: KarpenterNodeInstanceProfile-${CLUSTER_NAME}
-    capacityType: spot
+    capacityTypes: [spot]
```

Comment:
I don't think this is valid YAML. You need `"strings"` unless you're using `-` syntax

Reply:
works for me, but I'll make it uniform to match the rest.